ModKit/Utility/Dictionary/SerializableDictionary.cs

```diff
@@ -6,18 +6,22 @@
 
 namespace ModKit.Utility {
     [XmlRoot("SerializableDictionary")]
-    public class SerializableDictionary<TKey, TValue> : Dictionary<TKey, TValue>, IXmlSerializable, IUpdatableSettings {
+    public class SerializableDictionary<TKey, TValue> : Dictionary<TKey, TValue>, IXmlSerializable, IUpdatableSettings
```

Comment:
I did a thing in this class that I ended up not needing. Apparently it auto-formatted it. If you want me to revert, that shouldn't be a big deal.

Reply:
Please don 't change formatting if you can avoid it.

Reply:
Alright. Will revert.